posts/2022-02-08-Rust-Survey-2021.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Rust Survey 2021 Results"
+author: The Rust Survey Team
+---
+
+# Blog Post
+
+Greetings Rustaceans!
+
+Another year has passed, and with it comes another annual Rust survey analysis! The survey was conducted in December 2021.
+
+We’d like to thank everyone who participated in this year’s survey with a special shout-out to those who helped translate the survey from English into other languages.
+
+Without further ado, let’s dive into the analysis!
+
+## A Global Community
+
+The Rust community continues to grow, with this survey having the largest number of complete survey responses (9354 respondents), exceeding last year's total by roughly 1500 responses.
+
+90% of respondents said that they use Rust for any purpose including learning (an all-time high!) with 5% stating they have in the past and 4% stating they have yet to use Rust.
```

Comment:
Can you say more about 'all time high' here -- it's not clear to me why the percentage of respondents *using* Rust being high is a thing to celebrate. Don't we expect a high metric here?

It looks like this is mirroring language from previous years, but it feels like an odd thing to call out here. The particular sentence structure may also suggest that /learning/ is what is at an all time high, which is presumably incorrect? "learning" also feels like it's a little dangling? Are they just new users to Rust, or does learning here mean "formal" education (e.g., a CS degree).

Reply:
I think this is fair. We called this out as a thing to celebrate in previous posts, but I agree it's not really worth calling out.

As for "learning" - this refers to learning Rust in any capacity. I think we can just remove that bit.